tools/generate_debug_config.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+
+# Borrowed from upstream: https://github.com/envoyproxy/envoy/blob/main/tools/vscode/generate_debug_config.py
```

Comment:
suggestion to change this to a permanent link?

Reply:
good call -- updated this here: https://github.com/solo-io/envoy-gloo/pull/318/commits/adb9557f7af9bb1707ca020b43386407b0fbc1df